program/index.md

```diff
@@ -4,7 +4,8 @@ title: Program
 titlecontent:
 ---
 
-State of the Map 2017 is taking place from **August 18th to 20th 2017** in Aizuwakamatsu, Japan. We are looking forward to designing a program with you - the OpenStreetMap community. Please <a href="https://blog.openstreetmap.org/2017/02/16/propose-your-session-to-state-of-the-map-2017/">submit your session proposals</a> for a chance to be included in the program. 
```

Comment:
State of the Map 2017 is taking place from **August 18th to 20th 2017** in Aizuwakamatsu, Japan. We are looking forward to designing a program with you - the OpenStreetMap community. Thank you for the great responses to our <a href="https://blog.openstreetmap.org/2017/02/16/propose-your-session-to-state-of-the-map-2017/">call for proposals</a> and <a href="https://blog.openstreetmap.org/2017/04/11/community-survey-for-the-state-of-the-map-2017-program/">community input</a>! The final selection of talks and the building of the detailed program are making good progress thanks to our volunteers in the program committee. Watch here for more information soon.